test/src/receiver/main.cpp

```diff
@@ -94,34 +135,40 @@ int main() {
     using logger_top = cadmium::logger::multilogger<log_messages, global_time>;
 
 
-    /*******************************************/
 
-
-
-    /********************************************/
-    /****** APPLICATION GENERATOR *******************/
-    /********************************************/
+    /**
+     * It gets the input file for thr receiver execution and

```

Comment:
1) You may consider saying what input is specified 
2) There are typo's in this comment (Please check it)

Reply:
solved